requirements.txt

```diff
@@ -28,8 +28,7 @@ numpy>=1.16.5, <1.23.0
 pandas~=1.2, <1.5.0
 # used as a the engine for parquet files by pandas
 # >=10 to resolve https://issues.apache.org/jira/browse/ARROW-16838 bug that is triggered by ingest (ML-3299)
-# < 11 since starting from 11 ParquetDataset is deprecated and ParquetDatasetV2 is used instead
-pyarrow>=10,<11
+pyarrow>=10.0
```

Comment:
I think we may want to consider leaving an upper bound of `<12`, because when a major version comes out, they could easily break compatibility, and mlrun would be instantaneously broken.